### ISSUE TITLE
turkic publish offline mode outputs some rubbish links

### ISSUE THREAD

**Issue by mbakthav** [cvondrick/vatic#28]
When there is more than 1 video in the vatic database (ascertained turkic list), the turkic publish --offline options spews out invalid URLs. For instance, in my case, I had 2 videos (2 separate identifiers) in the vatic database. The tool outputs:
http://localhost?id=1&hitId=offline
http://localhost?id=2&hitId=offline
http://localhost?id=3&hitId=offline
http://localhost?id=16&hitId=offline
http://localhost?id=17&hitId=offline

Of these, only the last 2 are valid. Trying to access the first 3 URLs in the browser just displays '"Downloading the video" 
  0%

**Comment by drewm1980** [comment]
I am also trying to figure out how to actually launch turkic for the first time, offline.

`turkic find`

returns a bunch of links like yours that lead to a "Server Error" page.

`turkic publish --offline` doesn't do anything as far as I can tell.

**Comment by aaman-ee** [comment]
#24

**Comment by aaman-ee** [comment]
The issue lies in the database structure. This happens when you had previously loaded videos and deleted them. The next uploaded video will not have in the url id=1 but the incremental number of the segments.
For example, id=1 stands for the first segment of the first video you upload. since id is key for the table "jobs" you cannot reset it but goes incrementally. 
Try manually το change the id number. I found mine in id=51, and works.
I will also try to find a better solution, since in the current structure it is too difficult to work with many videos in offline mode.

**Comment by zesoft23** [comment]
I agree with the above. I believe the error occurs when a video is loaded without the 'offline' keyword but published with the 'offline' keyword. When that video is deleted the urlid counter is not properly adjusted and continues to include that deleted video. The accurate analogue in the database is groupid under vatic.turkic_hits Those id's should still correspond to the urlid's.

**Comment by feiyang-nottingham** [comment]
my problem is that it sends out many links. But none of them works. 
I load the video only once. 
And I use the option "--offline" for loading the publishing. 
Do you have any suggestion? 

http://localhost/?id=85&hitId=offline
http://localhost/?id=86&hitId=offline
http://localhost/?id=87&hitId=offline
http://localhost/?id=88&hitId=offline
http://localhost/?id=89&hitId=offline
http://localhost/?id=90&hitId=offline
http://localhost/?id=91&hitId=offline
http://localhost/?id=92&hitId=offline
http://localhost/?id=93&hitId=offline
http://localhost/?id=94&hitId=offline
http://localhost/?id=95&hitId=offline
http://localhost/?id=96&hitId=offline
http://localhost/?id=97&hitId=offline
http://localhost/?id=98&hitId=offline
http://localhost/?id=99&hitId=offline
http://localhost/?id=100&hitId=offline
http://localhost/?id=101&hitId=offline
http://localhost/?id=102&hitId=offline

    @zesoft23 @aaman-ee
